posts/2023-10-24-broken-badges-and-23k-keywords.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: A tale of broken badges and 23,000 features
+author: Tobias Bieniek
+team: the crates.io team <https://www.rust-lang.org/governance/teams/crates-io>
+---
+
+Around mid-October of 2023 we, the crates.io team, were [notified](https://github.com/rust-lang/crates.io/issues/7269) by one of our users that a [shields.io](https://shields.io) badge for their crate stopped working. The issue reporter was kind enough to already debug the problem and figured out that the API request that shields.io sends to crates.io was most likely the problem. Here is a quote from the original issue:
+
+> This crate makes heavy use of feature flags which bloats the response payload of the API.
```

Comment:
```suggestion
> This crate makes heavy use of feature flags which bloat the response payload of the API.
```

Reply:
this is quoted from the original issue. maybe we should not modify it?

Reply:
Author of the original issue here!

I don't mind if you guys modify it, english is not my first language and if you prefer correctness of language I'm totally fine with it.